static/app/views/settings/dynamicSampling/projectsEditTable.tsx

```diff
@@ -151,23 +171,52 @@ export function ProjectsEditTable({isLoading: isLoadingProp, sampleCounts}: Prop
           />
         ) : (
           <Fragment>
-            <SamplingBreakdown
-              sampleCounts={sampleCounts}
-              sampleRates={breakdownSampleRates}
-            />
-            <Divider />
-            <ProjectedOrgRateWrapper>
-              {t('Projected Organization Rate')}
-              <div>
+            <BreakdownWrapper>
+              <SamplingBreakdown
+                sampleCounts={sampleCounts}
+                sampleRates={breakdownSampleRates}
+              />
+            </BreakdownWrapper>
+            <FieldGroup
+              label={t('Projected Organization Rate')}
+              help={t(
+                'An estimate of the organization-wide sample rate, based on the span volume in the selected time frame.'
+              )}
+              flexibleControlStateSize
+              alignRight
+            >
+              <InputWrapper>
                 <PercentInput
                   type="number"
-                  disabled={!hasAccess}
+                  disabled={!hasAccess || !isBulkEditEnabled}
                   size="sm"
                   onChange={handleOrgChange}
                   value={projectedOrgRate}
                 />
-              </div>
-            </ProjectedOrgRateWrapper>
+                <FlexRow>
+                  <PreviousValue>
+                    {initialOrgRate !== projectedOrgRate ? (
+                      t('previous: %f%%', initialOrgRate)
+                    ) : (
+                      // Placeholder char to prevent the line from collapsing
+                      <Fragment>&#x200b;</Fragment>
+                    )}
+                  </PreviousValue>
+                  {hasAccess && !isBulkEditEnabled && (
+                    <BulkEditButton
+                      size="zero"
+                      title={t(
+                        'Scale all your project rates up and down by adjusting the organization-wide target'
+                      )}
+                      priority="link"
+                      onClick={() => setIsBulkEditEnabled(true)}
+                    >
+                      {t('bulk edit')}
```

Comment:
Any better idea for the button label?
this kinda says what you can do after clicking it, but it may be too technical 🤔

Reply:
Something like "edit all" or "enable to edit all"

Also it is a binary thing, maybe a checkbox would be appropriate (non-UI expert here 😅 )